File: runtimejs/java/util/IteratorEnumeration.js
//load// java/util/Enumeration
//load// java/lang/Object
var java_util_IteratorEnumeration = _extendClass( java_lang_Object, {
    
    _1: function(iterator) {
        this.iterator = iterator;
        return this;
    },
    
    hasMoreElements: function() {
        return this.iterator.hasNext();
    },
    
    nextElement: function() {  
        return this.iterator.next();
    },
    
}, "java_util_IteratorEnumeration", [java_util_Enumeration]);

File: runtimecs/java/util/IteratorEnumeration.cs
namespace java.util 
{
	public class IteratorEnumeration : Enumeration
	{
        private Iterator iterator;
        
        public IteratorEnumeration(Iterator i) {
            iterator = i;
        }
        
        public virtual bool hasMoreElements() {
            return iterator.hasNext();
        }
        
        public virtual System.Object nextElement() {
            return iterator.next();
        }
    }        
}
        
